Runtimes/Core/CMakeLists.txt

```diff
@@ -193,6 +193,7 @@ add_compile_options(
   "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-enable-experimental-feature Extern>"
   "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-enable-experimental-feature ValueGenerics>"
   "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-enable-experimental-feature AddressableParameters>"
+  "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-enable-experimental-feature AddressableTypes>"
```

Comment:
Is this needed on all Swift libraries (concurrency, swiftCore, and SwiftOnoneSupport) or just libswiftCore?

Reply:
For now it's just libswiftCore.

Reply:
We could put the flag just on the `swiftCore` target in `core/` to make it clear that it's the only target that needs it, but since it's an experimental-feature flag that will eventually be applied everywhere once the compiler enables it fully, I am not going to block the patch on it.

Reply:
I asked @jckarter what he'd prefer to do here, and I'll move AddressableParameters and AddressableTypes to the libswiftCore module config.